pkg/cmd/apiservicecabundle/cmd.go

```diff
@@ -81,9 +82,9 @@ func (o *ControllerCommandOptions) StartController() error {
 		return fmt.Errorf("unexpected config: %T", uncastConfig)
 	}
 
-	return controllercmd.NewController(componentName, (&apiservicecabundle.APIServiceCABundleInjectorOptions{Config: config}).RunAPIServiceCABundleInjector).
+	opts := &apiservicecabundle.APIServiceCABundleInjectorOptions{Config: config, LeaderElection: configv1.LeaderElection{}}
+	return controllercmd.NewController(componentName, opts.RunAPIServiceCABundleInjector).
 		WithKubeConfigFile(o.basicFlags.KubeConfigFile, nil).
-		// TODO we can update the API with leader election
-		//WithLeaderElection(config.LeaderElection, "", componentName+"-lock").
+		WithLeaderElection(opts.LeaderElection, "", componentName+"-lock").
```

Comment:
We probably want to specify `defaultNamespace` to make it easier to run this outside of a pod.

Reply:
(Applies to all `WithLeaderElection` blocks)